skll/data/writers.py

```diff
@@ -762,7 +708,7 @@ def _sanitize(name):
 
     def _write_line(self, id_, label_, feat_dict, output_file):
         """
-        Write the current line in the file in MegaM format.
+        Write the current line in the file in given format.
```

Comment:
To match the other writers, this should be:

```suggestion
        Write the current line in the file in this Writer's format.
```